### DISCUSSION TITLE
How can we add a tray icon?

### DISCUSSION THREAD

**Discussion by alvivar** [emilk/egui#2103] [Q&A]
How can we add a tray icon?

The idea is to create an application that has an icon on the taskbar, near the clock, the tray icon. This way you can use that icon to show/hide the screen of your app. Take the discord app as an example.

![image](https://user-images.githubusercontent.com/194978/193534777-52c83dcd-31f6-497d-8411-0a30b448485d.png)

Edit: Tray icon is a better description of the feature.

**Comment by yunfengwangluo** [comment]
这应该需要 用 windows API

**Comment by MrEnder0** [comment]
is there any specific libs for this?

**Comment by mokurin000** [comment]
tauri/`tray-icon`, which depends on GTK3
